### PR TITLE
Create session-replay.md

### PR DESCRIPTION
**Pull request by djmixpanel**
Creating session replay docs page. Broken image links, waiting on images PR to be accepted

https://github.com/mixpanel/docs/pull/1046

### CODE REVIEW

File: pages/docs/features/session-replay.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+![Untitled](https://prod-files-secure.s3.us-west-2.amazonaws.com/675c76ea-9344-43f8-957f-719aaedf54ce/506375a2-294e-4a09-a122-ad4201f88437/Untitled.png)
+
+**From Mixpanel reports**
+
+Click any point on a chart for Event and Funnel metrics, and select ‘View Replays’ to view replays that show that event being fired. From here, you will be taken to our Replay Player. 
+
+![Untitled](https://prod-files-secure.s3.us-west-2.amazonaws.com/675c76ea-9344-43f8-957f-719aaedf54ce/d2fa8f92-6058-4ade-bee1-cff3e0009790/Untitled.png)
+
+*Note: reports entry point is currently supported for Event and Funnel metric types only. Currently this excludes users, cohorts, retention reports, and flows reports.*
```
Comment (argenisf) [inline pages/docs/features/session-replay.md]: What about Insights? we phrase it as only in Events and Funnel, and then we mention the others it's excluded on, but not Insights. Perhaps we should just leave it at it's only supported in the Events and Funnels report, and say it's not supported in other reports yet (not to need to mention each one).

File: pages/docs/features/session-replay.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+- Expand the player to full-screen
+- Copy a URL to share with your teammates
+- Change the playback speed
+- Automatically skip periods of the replay where user is inactive
+
+The Replay Feed on the left of the player also allows you to:
+
+- sort replays by recency
+- search for replays by user name
```
Comment (argenisf) [inline pages/docs/features/session-replay.md]: what does this mean? is it bey the $name property? the $first_name and $last_name ones? you have to have a $username property?

File: pages/docs/features/session-replay.md (hunk 1 above)
Comment (djmixpanel) [reply pages/docs/features/session-replay.md]: Yes, Insights! I can reword, I think this means:

_*Note: reports entry point is currently supported with Events-based metrics inside Insights and Funnels reports._ 

I didnt love this cause like everything is Events in mixpanel, but Replays only works for Events, not Insights reports where the metric is Cohorts or Users based

File: pages/docs/features/session-replay.md (hunk 2 above)
Comment (djmixpanel) [reply pages/docs/features/session-replay.md]: I think $first_name and $last_name, or email if these not defined! - CC @carlosdanieltt please correct me if I'm wrong

Comment (carlosdanieltt) [reply pages/docs/features/session-replay.md]: Yep, we grab their first and last name if available.  Email and "User Profile" are the other fallbacks.

Comment (carlosdanieltt) [reply pages/docs/features/session-replay.md]: Also, @djmixpanel searching by replay date and user distinct id is also supported.

Comment (djmixpanel) [reply pages/docs/features/session-replay.md]: Thanks @carlosdanieltt !

**Review by ishamehramixpanel — APPROVED** [review]

**Pull request merged**